src/custom/components/swap/EthWethWrap/index.tsx

```diff
@@ -137,9 +134,7 @@ export default function EthWethWrap({ account, native, userInput, wrapped, wrapC
       <WarningWrapper>
         <AlertTriangle size={25} />
         <div>
-          <span>
-            To sell {nativeSymbol}, first wrap or switch to {wrappedSymbol}
-          </span>
+          Wrap your {nativeSymbol} first or switch to {wrappedSymbol}!
```

Comment:
not sure the text should be changed in this PR.

I don't like the `!` it's too intense, i would just remove it @anxolin @alfetopito @annamsgeorge

Reply:
Was changed, as the text should be bigger than it was. As a result, trimmed the text down in order to achieve that.

Reply:
Can remove the ! if too intense. Personally good with it.